zyngui/zynthian_gui_fileselector.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#******************************************************************************
+# ZYNTHIAN PROJECT: Zynthian GUI
+# 
+# Zynthian GUI Step-Sequencer Class
+# 
+# Copyright (C) 2015-2020 Fernando Moyano <jofemodo@zynthian.org>

```

Comment:
I can't remember to have contributed to this piece of code, but if you mean i "inspired" the code in some way, you can leave the line to rest  ;-)